### ISSUE TITLE
Toolbar button tooltips should also be visible for keyboard users

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#5581]
## 📝 Provide detailed reproduction steps (if any)

Currently button tooltips are not visible for keyboard users.

1. Open an [example editor with toolbar](https://ckeditor.com/docs/ckeditor5/12.4.0/features/basic-styles.html).
2. Press `alt + f10` to focus the toolbar.

### ✔️ Expected result

The toolbar button tooltip should be shown, just like when the mouse cursor is over the option.

![expected](https://user-images.githubusercontent.com/5353898/66742285-9a5fcf80-ee77-11e9-990f-6f3503069670.png)

### ❌ Actual result

No tooltip is visible.

![current situation](https://user-images.githubusercontent.com/5353898/66742300-a21f7400-ee77-11e9-91da-1068d85a9bec.png)

## 📃 Other details

* CKEditor version: 12.4.0

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
For anyone stumbling upon this problem, you can temporarily add following CSS:

```css
div.ck-toolbar  .ck.ck-button:focus .ck-tooltip {
	display: block;
	visibility: visible;
	opacity: 1;
}
```

To get around this problem until this issue is resolved. You can see it here https://codepen.io/mlewand/pen/wvvGvjM

**Comment by wimleers** [comment]
Yay, #11869 fixed one of the aspects of https://www.drupal.org/project/drupal/issues/3283800 🥳 🚀

**Comment by Reinmar** [comment]
Followup topic on why we need these tooltips and whether they are good: #12066.

**Comment by Reinmar** [comment]
We'll need to reopen this ticket (see the relevant discussion in #12066).

We'll try to come up with a better, more sophisticated solution.

**Comment by Reinmar** [comment]
Followup ticket to bring back a better implementation: #12067.

Ticket for reverting the original fix: #12068.
